docs: update WebDriver BiDi documentation (#11536)
puppeteer/puppeteer: docs/webdriver-bidi.md

> # Experimental WebDriver BiDi Support

Author: ```suggestion
# Experimental WebDriver BiDi support
```

[WebDriver BiDi](https://w3c.github.io/webdriver-bidi/) is a new cross-browser

> automation protocol currently under development, aiming to combine the best of both WebDriver “Classic” and CDP. WebDriver BiDi promises bi-directional communication, making it fast by default, and it comes packed with low-level control. Dive deeper with these resources:

Reviewer: ```suggestion
automation protocol currently under development, aiming to combine the best of both WebDriver “Classic” and CDP. WebDriver BiDi promises bi-directional communication, making it fast by default, and it comes packed with low-level control.

See also:
```
